src/lib.rs

```diff
@@ -165,7 +165,9 @@ pub async fn start_lemmy_server() -> Result<(), LemmyError> {
     };
 
     App::new()
-      .wrap(middleware::Logger::default())
+      .wrap(middleware::Logger::new(
+        "%{r}a %r %s %b %{Referer}i %{User-Agent}i %T",
+      ))
```

Comment:
Can you add a comment to describe what this does? Basically what you wrote in the PR.